src/main/resources/Messages.properties

```diff
@@ -1570,6 +1572,7 @@ environment.has.plot.data=All observations and/or fieldmap for this environment
 study.should.have.at.least.one.environment=Study should have at least one environment.
 
 crosses.and.selections.no.germplasm.error=Please select at least one germplasm.
+study.observations.changing.plot.entry.warning=If you change the entry all the information related to these plots (including observations) will be now referenced to the new entry. Are you sure you want to proceed?
```

Comment:
Improve grammar:
If you change the entry**,** all the information related to these plots (including observations) will **now refer to** the new entry. Are you sure you want to proceed?